installing/installing_vmc/installing-restricted-networks-vmc.adoc

```diff
@@ -38,6 +38,8 @@ include::modules/cluster-entitlements.adoc[leveloffset=+1]
 
 include::modules/installation-vsphere-infrastructure.adoc[leveloffset=+1]
 
+include::modules/installation-vsphere-installer-network-requirements.adoc[leveloffset=+1]
```

Comment:
You could include this module in the same location within each of the assemblies, or as close to the same location as possible. At present, the module positioning does not seem consistent across the assemblies. Introducing consistency in the structure will help with findability.

This applies to all of the assemblies updated in this PR, but I won't mention this comment in all of them.

Reply:
Addressed